packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -410,6 +410,14 @@ export class SwapQuoter {
         return whitelistedApiKeys.includes(apiKey);
     }
 
+    private _isTxOriginBlacklisted(txOrigin: string | undefined): boolean {
+        if (!txOrigin) {
+            return false;
+        }
+        const blacklistedTxOrigins = this._rfqtOptions ? this._rfqtOptions.txOriginBlacklist : new Set();
```

Comment:
Without having to create a `new Set` every time this fn is called, can we change something like
```
if (this._rfqtOptions && this._rfqtOptions.txOriginBlacklist && this._rfqtOptions.txOriginBlacklist.has(txOrigin.toLowerCase())) {
     return true;
}
return false;
```

this is optional, if it makes sense to you